package.json

```diff
@@ -60,8 +65,9 @@
     "rollup-plugin-replace": "^1.1.1",
     "rollup-watch": "^2.4.0",
     "selenium-server": "^2.53.1",
+    "typescript": "^2.0.3",
     "uglify-js": "^2.7.0",
-    "vue": "^2.0.0-rc.6",
+    "vue": "vuejs/vue#eef05ac9e93a2b841caf576e5bc3e85d28472594",
```

Comment:
Yeah, exactly. But current published typings on npm is outdated because `ComponentOptions` is now generic type...
Should we wait until next version of Vue is published?